libcodechecker/server/client_auth_handler.py

```diff
@@ -42,13 +43,23 @@ def getAuthParameters(self):
                                         True))
 
     @timeit
+    def getLoggedInUser(self):
+        if self.__auth_session:
+            return self.__auth_session.user
+        else:
+            return ""
```

Comment:
Unfortunately the javascript thrift parser throws an exception when we return None from python for function with string return type. So I will leave it like this.